pom.xml

```diff
@@ -156,22 +156,42 @@
               <goal>shade</goal>
             </goals>
             <configuration>
-             <createDependencyReducedPom>false</createDependencyReducedPom>
-             <artifactSet>
+              <createDependencyReducedPom>true</createDependencyReducedPom>
```

Comment:
I think this was an oversight when it was added but it is better to use the dependency reduced pom as the classes we shade are now in the s3Proxy jar so no need to pull them in via the pom dependencies.
The `promoteTransitiveDependencies` makes sure and transitive deps are promoted and added to the dependency reduced pom.